openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
This is my fault; I wasn't clear enough. My concern is that a name like `Public` is a bit vague; `PublicAccess` would be better. In addition, `FlavorType` and `FlavorAccess` are OK, but we're in the `flavors` package, so the functions and types are assumed to relate to flavors. Here's a suggestion to make the code a bit more readable:
```go
type AccessType string

const (
  PublicAccess AccessType = "true"
  PrivateAccess AccessType = "false"
  AllAccess AccessType = "None"
)
```
And it would be used like:
```go
listOpts := flavors.listOpts{
  FlavorAccess: flavors.PublicAccess,
}
```

Note that the "Type" part of "AccessType" is not because it's a Go type (which it is), but because it is the type of access granted.

@jtopjian, @mfranc213 questions or thoughts about that?